### PR TITLE
CASMTRIAGE-7372 -- 1.6

### PR DESCRIPTION
**Pull request by jpdavis-prof**
# Description

Remove references to cps-etcd and cm-pm.
https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-7372

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

### CODE REVIEW

File: operations/node_management/Add_a_Standard_Rack_Node.md [hunk 1]
```diff
@@ -526,8 +526,7 @@ If adding Foxconn (Paradise) nodes to the system, follow the [Replacing Foxconn
 
 **These troubleshooting steps are applicable when DVS is operating over the NMN network, and not the HSN network.**
 
-There should be a `cray-cps` pod (the broker), three `cray-cps-etcd` pods and their waiter, and at least one `cray-cps-cm-pm` pod.
-Usually there are two `cray-cps-cm-pm` pods, one on `ncn-w002` and one on `ncn-w003` and other worker nodes.
+There should be one or more `cray-cps` pods.
```
Comment (leliasen-hpe) [inline operations/node_management/Add_a_Standard_Rack_Node.md:529]: Double check with COS/DVS if this is correct

File: operations/node_management/Add_a_Standard_Rack_Node.md [hunk 2]
```diff
@@ -539,15 +538,9 @@ Usually there are two `cray-cps-cm-pm` pods, one on `ncn-w002` and one on `ncn-w
 
     ```text
     services   cray-cps-75cffc4b94-j9qzf    2/2  Running   0   42h 10.40.0.57  ncn-w001
```
Comment (leliasen-hpe) [inline operations/node_management/Add_a_Standard_Rack_Node.md:540]: Check if this is correct

File: operations/node_management/Add_a_Standard_Rack_Node.md (hunk 1 above)
Comment (jpdavis-prof) [reply operations/node_management/Add_a_Standard_Rack_Node.md:529]: I check with Kazuya and verified it on a running system.

File: operations/node_management/Add_a_Standard_Rack_Node.md (hunk 2 above)
Comment (jpdavis-prof) [reply operations/node_management/Add_a_Standard_Rack_Node.md:540]: I check with Kazuya and verified it on a running system.

**Review by nrockershousen — APPROVED** [review]

**Review by mtupitsyn — APPROVED** [review]

**Pull request merged**